src/content/en/fundamentals/performance/rail.md

```diff
@@ -1,171 +1,350 @@
+  or 4G connection, in reality the *effective connection speed* is often significantly slower,
+  due to packet loss and network variance.
+* Focus on optimizing the [Critical Rendering Path][CRP] to unblock rendering.
+* You don't have to load everything in under 5 seconds to produce the perception of a complete
+  load. Enable progressive rendering and do some work in the background. Defer non-essential
+  loads to periods of idle time. See [Website Performance Optimization][Udacity].
+* Recognize the factors that affect page load performance:
+    * Network speed and latency
+    * Hardware (slower CPUs, for example)
+    * Cache eviction
+    * Differences in L2/L3 caching
+    * Parsing JavaScript
+
+[ME17]: https://www.gsma.com/mobileeconomy/
+[WPT]: https://www.webpagetest.org/easy
+[CRP]: /web/fundamentals/performance/critical-rendering-path/
+[Udacity]: https://www.udacity.com/course/website-performance-optimization--ud884
+[Budgets]: https://infrequently.org/2017/10/can-you-afford-it-real-world-web-performance-budgets/
+
+## Tools for measuring RAIL {: #tools }
+
+There are a few tools to help you automate your RAIL measurements. Which one you use depends
+on what type of information you need, and what type of workflow you prefer:
+
+* [**Chrome DevTools**](#devtools). The developer tools built into Google Chrome. Provides
+  in-depth analysis on everything that happens while your page loads or runs.
+* [**Lighthouse**](#lighthouse). Available in Chrome DevTools, as a Chrome Extension, as a
+  Node.js module, and within WebPageTest.
+  You give it a URL, it simulates a mid-range device with a slow 3G connection, runs a
+  series of audits on the page, and then gives you a report on load performance, as well
+  as suggestions on how to improve. Also provides audits to improve
+  accessibility, make the page easier to maintain, qualify as a Progressive Web App, and more.
+* [**WebPageTest**](#webpagetest). Available at
+  [webpagetest.org/easy](https://webpagetest.org/easy). You give
+  it a URL, it loads the page on a real Moto G4 device with a slow 3G connection, and then gives
+  you a detailed report on the page's load performance. You can also configure it to include a
+  Lighthouse audit.
+
+The sections below give you more information on how to use each tool to measure RAIL.
+
+### Chrome DevTools {: #devtools }
+
+The **Performance** panel is the main place to analyze your RAIL metrics. See [Get Started With
+Analyzing Runtime Performance][cdt-runtime] to get familiar with the **Performance** panel UI.
+The workflow and UI for analyzing load performance is mostly the same, the only difference is how
+you start and stop the recording. See [Record load performance][cdt-load].
+
+[cdt-runtime]: /web/tools/chrome-devtools/evaluate-performance/
+[cdt-load]: /web/tools/chrome-devtools/evaluate-performance/reference#record-load
+
+The following DevTools features are especially relevant:
+
```

Comment:
Admittedly I could probably organize this section a little better, but I think it's a good start.

Reply:
I'm a fan of this idea of explicitly calling out features that are relevant to RAIL here across our tooling better. I vote for filing an issue to rethink how we can organize this section better but moving ahead with what we've got here for now.